examples/http-client-payload-constraint-validation/http_client_payload_constraint_validation.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Payload constraint validation
 
-Through client payload constraint validation, the response payload can be validated according to the defined constraints. The constraint validation happens along with the data binding step. If the validation fails, a `http:PayloadValidationError` will be returned with the validation details.
+The client-level response data binding also validates the payload against any constraints defined on the response payload type. If the validation fails, `http:PayloadValidationError` will be returned with the validation error message. Use this to directly validate the response payload with respect to the defined constraints.
```

Comment:
How about a description like the below?

The Ballerina constraint package allows you to add additional constraints to the response payload. `http:Client` uses the constraint package to validate the payload against the given constraints. Validation happens soon after the successful data-binding of the payload. Constraints can be added to a given data-type using different annotations. If the validation fails, `http:PayloadValidationError` will be returned with the validation error message. Use this to validate the response payload as the application program receives it, which allows you to guard against unnecessary processing and malicious payloads.

Reply:
Changed the description of both BBEs in [a5ba91f](https://github.com/ballerina-platform/ballerina-distribution/pull/3743/commits/a5ba91fab573537e4d7a0cd632d9b7722f3c19bf)
Please review